<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_instance_generator</class>
 <widget class="QDialog" name="dialog_instance_generator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>460</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>291</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Instance type</string>
   </property>
   <widget class="QRadioButton" name="radioButton_small">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Small</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_medium">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Medium</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_large">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Large</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_custom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Custom settings</string>
   </property>
   <widget class="QSpinBox" name="spinBox_nbtimeslots">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of timeslots</string>
    </property>
    <property name="buddy">
     <cstring>spinBox_nbtimeslots</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of events per timeslot</string>
    </property>
    <property name="buddy">
     <cstring>spinBox_nbeventspertimeslot</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min nb people eventgroup</string>
    </property>
    <property name="buddy">
     <cstring>spinBox_minnbpeople</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max nb people eventgroup</string>
    </property>
    <property name="buddy">
     <cstring>spinBox_maxnbpeople</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_nbeventspertimeslot">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_minnbpeople">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>250</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_maxnbpeople">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>250</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Instance name</string>
   </property>
   <property name="buddy">
    <cstring>lineedit_instancename</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_instancename">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_nblocations">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of locations</string>
   </property>
   <property name="buddy">
    <cstring>spinBox_nblocations</cstring>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialog_instance_generator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialog_instance_generator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
